build.sbt

```diff
@@ -11,10 +11,13 @@ lazy val `latis-hapi` = (project in file("."))
     name := "latis-hapi",
     libraryDependencies ++= Seq(
       "org.http4s"    %% "http4s-blaze-server" % http4sVersion,
+      "org.http4s"    %% "http4s-circe"        % http4sVersion,
       "org.http4s"    %% "http4s-dsl"          % http4sVersion,
       "ch.qos.logback" % "logback-classic"     % "1.2.3" % Runtime,
-      "com.lihaoyi"   %% "scalatags"           % "0.6.7"
-    )
+      "com.lihaoyi"   %% "scalatags"           % "0.6.7",
+      "org.scalatest" %% "scalatest"           % "3.0.5" % Test
+    ),
+    Test / logBuffered := false
```

Comment:
[ScalaTest recommends this setting](http://www.scalatest.org/user_guide/using_scalatest_with_sbt) because they do their own buffering.